src/realm/dictionary.cpp

```diff
@@ -44,6 +47,11 @@ void validate_key_value(const Mixed& key)
 
 // Dummy cluster to be used if dictionary has no cluster created and an iterator is requested
 static DictionaryClusterTree dummy_cluster(nullptr, type_Int, Allocator::get_default(), 0);
+static std::mt19937_64 generator(std::chrono::system_clock::now().time_since_epoch().count());
+
+#ifdef REALM_DEBUG
+uint64_t Dictionary::s_hash_mask = 0x7FFFFFFFFFFFFFFFULL;
```

Comment:
Can we add a comment about what exactly this mask is for? Is the significance of 0x7FFFFFFFFFFFFFFFULL that it masks an int64_t.